packages/components/NetworkSelector/CustomeNetworkSelector.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useRef } from "react";
```

Comment:
CustomeNetworkSelector.tsx => Maybe a type error in the file/component names ?

Reply:
Okay, I will fix it.